src/sentry/static/sentry/app/utils/withExperiment.tsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 
 import ConfigStore from 'app/stores/configStore';
 import {Organization} from 'app/types';
-import {getDisplayName} from 'app/utils/environment';
 import {experimentConfig, unassignedValue} from 'app/data/experimentConfig';
+import getDisplayName from 'app/utils/getDisplayName';
```

Comment:
Seems to be a case of mistaken identity between 2 functions.

Reply:
Oops, that’s my b